CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ We use _breaking :warning:_ to mark changes that are not backward compatible (re
 - [#4107](https://github.com/thanos-io/thanos/pull/4107) Store: `LabelNames` and `LabelValues` now support label matchers.
 - [#4171](https://github.com/thanos-io/thanos/pull/4171) Docker: Busybox image updated to latest (1.33.1)
 
+- [#4176](https://github.com/thanos-io/thanos/pull/4176) Query API: Adds optional `Stats param`
```

Comment:
Please remove the blank line above and add one blank line below.
`Adds optional `Stats param` to return stats for query APIs` would be better.